src/main/webapp/WEB-INF/tags/admin/navBar.tag

```diff
@@ -22,9 +22,10 @@
                     <a href="<%=Const.ActionURIs.ADMIN_HOME_PAGE%>">Create Instructor</a>
                 </li>
                 
+                <!-- The link to Account Management Page will be hidden until it's scalable.
```

Comment:
@bamboo3250, let's just remove the link instead of just commenting it out. If the link isn't added back in the very near future, in all likelihood, when the person adds the link in future, he/she won't notice, or will just ignore, the commented out code.